modules/rbac-resource-definitions.adoc

```diff
@@ -6,15 +6,21 @@
 = Resource definitions
 
 [role="_abstract"]
-{product-title} includes multiple resources.
+{product-title} includes many resources.
 The following table lists the resources and describes the actions that users can perform with the `read` or `write` permission.
 
+[NOTE]
+====
+* To prevent privilege escalation, when you create a new token, your role's permissions limit you. For example, if you only have `read` permission for the Integration resource, you cannot create a token with `write` permission.
+* If you want a custom role to create tokens for other users to use, you must assign the required permissions to that custom role.
+====
```

Comment:
We don't want to use the snippet here?

Reply:
Thank you @mburke5678 I wasn't sure if we can use snippets in modules. I've included it.